Bali/src/main/java/GetNameServlet.java

```diff
@@ -0,0 +1,19 @@
+import java.io.*;
+import javax.servlet.http.*;
+
+public class GetNameServlet extends HttpServlet {
```

Comment:
I'm experiencing a bitterfit of deja vu(( Why these files are here again?
Please remember to always offer for a review a minimal changeset. The precise pieces of code you need a review for. This relates to pull requests in general (too much old code spoils git history, makes it impossible to track changes and bugs)